cadence/contracts/auth/AxelarAuthWeighted.cdc

```diff
@@ -46,9 +46,17 @@ access(all) contract AxelarAuthWeighted {
   /***********************\
   |* Owner Functionality *|
   \***********************/
-  access(account) fun transferOperatorship(params: TransferOperatorshipParams) {
-    self._transferOperatorship(newOperators: params.newOperators, newWeights: params.newWeights, newThreshold: params.newThreshold)
-    emit OperatorshipTransferred(newOperators: params.newOperators, newWeights: params.newWeights, newThreshold: params.newThreshold)
+  access(account) fun transferOperatorship(
```

Comment:
Just confirming - do we want this to be a no-op on failure or would we rather panic here?